src/helm/clients/together_client.py

```diff
@@ -346,13 +346,29 @@ def convert_to_raw_chat_request(self, request: Request) -> TogetherRawChatReques
                     message_contents.append({"type": "text", "text": media_object.text})
                 else:
                     raise ValueError(f"Unrecognized MediaObject type {media_object.type}")
+
+            if request.model_engine in ["llama-3.2-11b-vision-instruct-turbo", "llama-3.2-90b-vision-instruct-turbo"]:
+                # Only supports a single input image. Remove the excess images.
```

Comment:
Do we have scenarios with multiple images? If so, you should at least log a warning if images get stripped. If not, you should raise an exception if trying to send multiple images to pixtral e.g.

```python
if len([message in message_contents if message["type"] == "image_url"]) > 1:
    raise ValueError("...")
```

Reply:
Yes, there are some scenarios that have multiple images. I will log a warning.

Reply:
Noted. There are two more problems with this block:

1. The images are already added above at lines 335-344, so you are double-adding the images here.
2. This logic should happen within the `for media_object in request.multimodal_prompt.media_objects` loop. By moving it outside, you are causing the image objects to be reordered after the text objects.

Basically, you should modify the block at lines 335-344 to include this first-image filtering logic.

Reply:
Note that I created a new list: `filtered_message_contents = []`. Also, at every step I'm adding some content (text or image). With the inner if-statement, I'm not adding subsequent images.

Reply:
Also, I'm not sure if I should do this filter in the first place. I think I'm just going to undo my change in this file altogether.